src/client/clientmap.cpp

```diff
@@ -143,6 +143,7 @@ void ClientMap::getBlocksInViewRange(v3s16 cam_pos_nodes,
 			p_nodes_max.Z / MAP_BLOCKSIZE + 1);
 }
 
+static const constexpr s16 UNLOAD_PADDING = MAP_BLOCKSIZE * 3;
```

Comment:
Why is this not just MAP_BLOCKSIZE?
The server will only allow one extra block of range to be sent anyway.

Reply:
I was thinking of a player roaming a small area, but maybe it's best left to the unload timeout to handle that. Reduced.